css/properties/contain.json

```diff
@@ -74,7 +74,9 @@
             "spec_url": "https://drafts.csswg.org/css-contain/#valdef-contain-style",
             "support": {
               "chrome": {
-                "version_added": "52"
+                "version_added": "52",
+                "partial_implementation": true,
```

Comment:
Thanks @foolip - I've removed partial implementation in https://github.com/mdn/browser-compat-data/pull/19978/commits/99c1f539ccfdf6afe75996e1b0b141d2d73f9566

There is only one relevant bug here, so leaving a single note with this.

Reply:
> I suspect these bugs aren't serious enough to affect most developers using `contain: style`, so I suggest no "partial_implementation" here.

If this is true, then please also remove partial_implementation for Safari. Seems unfair otherwise.

Reply:
done in b3ee18c6cc0012c677fd1943d0c321a753ddc983

Reply:
> I suspect these bugs aren't serious enough to affect most developers using `contain: style`, so I suggest no "partial_implementation" here.
> 
> For the notes, can you make it an array, one per bug, and use version numbers following this style?
> 
> https://github.com/mdn/browser-compat-data/blob/f16db66267d457a2349d1489544f28d606c897d1/css/properties/border-image-width.json#L9-L12

Hi @bsmth  / @foolip ,

This bug might not affect 90% of the developers, but, for the 10% of the developers who are affected (like myself), it might take us days (if not weeks) to locate the root cause of the problem (depending on the experience/seniority of the developers).

Isn't the spirit of the documentation to accurately document the compatibility of each browser?  

Thanks!